pkg/providers/aws/provider_redis.go

```diff
@@ -99,6 +101,34 @@ func (p *AWSRedisProvider) CreateRedis(ctx context.Context, r *v1alpha1.Redis) (
 	// setup aws elasticache cluster sdk session
 	cacheSvc, stsSvc := createAWSService(stratCfg, providerCreds)
 
+	// Retrieve service maintenance updates, create and export Prometheus metrics
+	output, err := cacheSvc.DescribeServiceUpdates(&elasticache.DescribeServiceUpdatesInput{})
+	if err != nil {
+		logrus.Info(fmt.Sprintf("elasticache serviceupdates error: %s:", err))
```

Comment:
We can return on this error
```
if err != nil {
     errMsg := "elasticache service updates error"
     return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)```
}

Once we return on this error, we can then remove the `else` statement